nucypher/blockchain/eth/actors.py

```diff
@@ -136,10 +141,15 @@ class Deployer(NucypherTokenActor):
         AdjudicatorDeployer,
         UserEscrowProxyDeployer,
     )
+    __interface_class = BlockchainDeployerInterface
 
-    contract_names = tuple(a.registry_contract_name for a in EthereumContractAgent.__subclasses__())
+    __secrets = ('staker_secret',
+                 'policy_secret',
+                 'user_escrow_proxy_secret',
+                 'adjudicator_secret')
+    Secrets = collections.namedtuple('Secrets', __secrets)
 
-    __interface_class = BlockchainDeployerInterface
+    contract_names = tuple(a.registry_contract_name for a in EthereumContractAgent.__subclasses__())
```

Comment:
Quasimodo in here.  Maybe this needs to be a `classmethod` on `EthereumContractAgent`?

Reply:
:heavy_check_mark: